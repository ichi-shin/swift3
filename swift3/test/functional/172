#!/bin/bash
# Copyright(c)2014 NTT corp. All Rights Reserved.
# DB1-1-4-1_3

. ./common
S3USER=tester
_s3_put /bucket

S3USER=tester2
_s3_put /bucket2

S3USER=tester
_s3_get / -H "hoge: hoge" -D - | _filter_curl xml
