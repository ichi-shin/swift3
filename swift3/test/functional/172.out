QA output created by 172
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester2 -- -X PUT http://SWIFT_HOST/bucket2... 200
> s3curl --id tester -- -X GET -H hoge: hoge -D - http://SWIFT_HOST/... 200
HTTP/1.1 200 OK
Content-Length: LENGTH
Content-Type: application/xml
Date: DATE
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

<?xml version='1.0' encoding='UTF-8'?>
<ListAllMyBucketsResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
  <Owner>
    <ID>TESTER</ID>
    <DisplayName>TESTER</DisplayName>
  </Owner>
  <Buckets>
    <Bucket>
      <Name>bucket</Name>
      <CreationDate>DATE</CreationDate>
    </Bucket>
  </Buckets>
</ListAllMyBucketsResult>
